guides/CreatingBackends.md

```diff
@@ -1,79 +0,0 @@
-# Creating Backends
```

Comment:
why removing this ? we need a way let people know how to use the backend even with your new API? or is this moved somewhere else I missed?

Reply:
Yes, it will be documented in the behaviors, similar to DBConnection.